Framework/include/QualityControl/CheckRunner.h

```diff
@@ -37,6 +37,7 @@
 #include "QualityControl/MonitorObject.h"
 #include "QualityControl/QcInfoLogger.h"
 #include "QualityControl/Check.h"
+#include "QualityControl/ServiceDiscovery.h"
```

Comment:
Could you try forward-declaring `ServiceDiscovery` here? I see that we need only the pointer in this header. 
```suggestion
namespace o2::quality_control::core
{
class ServiceDiscovery;
}
```
Then .cxx file of course needs the include statement.

Reply:
sure